src/components/types.ts

```diff
@@ -238,3 +238,21 @@ export type MultimodalInputProps = TextInputProps &
     | 'filePreviewsContainer'
     | 'errorMessagesContainer'
   >
+
+export interface ConversationProps {
+  /** WebSocket connection to send and receive messages to and from a backend. This value will be set automatically if the component is rendered with `ElementRenderer` or `MessageSpace`. */
+  ws: WebSocketClient
+  /** Id of the current user. This value will be set automatically if the component is rendered with `ElementRenderer` or `MessageSpace`. */
+  sender: string
+  /** Id of the current conversation. This value will be set automatically if the component is rendered with `ElementRenderer` or `MessageSpace`. */
+  conversationId: string
+  /** ID of the message. This value will be set automatically if the component is rendered with `ElementRenderer` or `MessageSpace`. */
```

Comment:
nit: In some places you use `Id` whereas others use `ID`

Reply:
Will go over all files and raise a follow up pr for consistency